src/users/api/views.py

```diff
@@ -0,0 +1,35 @@
+from django.contrib.auth import get_user_model
+from django.http import JsonResponse
+from django.templatetags.static import static
+from rest_framework.decorators import api_view, authentication_classes, permission_classes
+from rest_framework.permissions import IsAuthenticated
+
+from core.authentication import TokenAuthentication
+
+ALLOWED_ROLES = {"Program", "Reviewer", "Sponsor team"}
+DEFAULT_PHOTO_URL = static('images/default_head.png')
+
+
+User = get_user_model()
+
+@api_view(['GET'])
+@authentication_classes([TokenAuthentication])
+@permission_classes([IsAuthenticated])
+def user_list(request):
+    role = request.GET.get('role')
+    if role and role not in ALLOWED_ROLES:
+        return JsonResponse({'detail': 'Invalid role.'}, status=400)
```

Comment:
不應有可以 expose 所有 user data 的 API
```suggestion
    if not role or role not in ALLOWED_ROLES:
        return JsonResponse({'detail': 'role is not given or invalid.'}, status=400)
```